pkgs/applications/office/ledger/default.nix

```diff
@@ -12,11 +12,9 @@ stdenv.mkDerivation rec {
     sha256 = "0x6jxwss3wwzbzlwmnwb8yzjk8f9wfawif4f1b74z2qg6hc4r7f6";
   };
 
-  outputs = [ "out" "dev" ];
```

Comment:
Why did you remove this?

Reply:
The changes to ledger were in my shadow and I didn't mean to add them to this PR; apologies!  Will update.

(The `dev` output happens to be both broken and also unused, though, which is why I was looking at patching/fixing/changing it somehow.)

Reply:
The output was really empty? Then it can be removed.